apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/TransformerUtilsV2.java

```diff
@@ -3478,8 +3490,11 @@ static ItemComponent mapEobCommonItemRevenue(
             C4BBAdjudication.NONCOVERED,
             nonCoveredChargeAmount));
 
-    // REV_CNTR_UNIT_CNT => ExplanationOfBenefit.item.quantity
-    item.setQuantity(new SimpleQuantity().setValue(unitCount));
+    // REV_CNTR_NDC_QTY => ExplanationOfBenefit.item.quantity
+    Extension drugQuantityExtension =
+    createExtensionQuantity(CcwCodebookVariable.REV_CNTR_NDC_QTY, nationalDrugCodeQuantity);
+    Quantity drugQuantity = (Quantity) drugQuantityExtension.getValue();
+    item.setQuantity(drugQuantity);
```

Comment:
Just curious....was there any reason you oyt the code here instead of where the TODO comment was located?

Reply:
No. I missed that TODO comment. Ill put the code there instead.